data-reporting-service/observation-service/src/main/java/gov/cdc/etldatapipeline/observation/config/LogDynamicFileAppenderConfig.java

```diff
@@ -0,0 +1,62 @@
+package gov.cdc.etldatapipeline.observation.config;
+
+import ch.qos.logback.core.FileAppender;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class LogDynamicFileAppenderConfig<E> extends FileAppender<E> {
+
+    private String logFilePath;
+
+    /**
+     * Helper method used by Logback
+     * Reading logFilePatch tag from dlt-logback.xml and return value
+     * */
+    public void setLogFilePath(String logFilePath) {
+        this.logFilePath = logFilePath;
+    }
+
+    /**
+     * Purpose: Dynamically create log file if not exist
+     * */
+    @Override
+    public void start() {
+        if (logFilePath == null) {
+            addError("Log file path is not configured");
+            return;
+        }
+
+        if (logFilePath.contains("%d{")) {
+            SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
+            dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
@rshanmugam-enquizit Do not use UTC stamp instead use local

Reply:
Done